File: Shift.Infrastructure/Models/ViewModels/Auth/GraduateRegisterVM.cs
using System;

namespace Shift.Infrastructure.Models.ViewModels.Auth
{
	using Shift.DAL.Models.University;

	public class GraduateRegisterVM
	{
		public UserRegisterVM User { get; set; }

		public EducationForm EducationForm { get; set; }

		public int? SpecialtyId { get; set; }

		public int AdviserId { get; set; }

		public DateTime? StartEducationDate { get; set; }
		public DateTime? FinishEducationDate { get; set; }
	}
}

File: Shift.Web/ClientApp/src/app/infrastracture/entities/auth/GraduateRegisterVM.ts
import { UserRegisterVM } from "./UserRegisterVM";

export class GraduateRegisterVM {
    public User: UserRegisterVM;

    public  EducationForm: number;

    public SpecialtyId: number;
    public AdviserId: number;

    public StartEducationDate: Date;
    public FinishEducationDate: Date;
}